mkdocs/commands/gh_deploy.py

```diff
@@ -14,8 +14,12 @@
 
 
 def _is_cwd_git_repo():
-    proc = subprocess.Popen(['git', 'rev-parse', '--is-inside-work-tree'],
+    try:
+        proc = subprocess.Popen(['git', 'rev-parse', '--is-inside-work-tree'],
                             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Flake8 is [complaining][1] about the indentation here.

You either need to indent line 19 so it lines up with the first character and the opening `(` on line 18, or break it up like this:

```python
proc = subprocess.Popen(
    ['git', 'rev-parse', '--is-inside-work-tree'],
    stdout=subprocess.PIPE, 
    stderr=subprocess.PIPE
)
```

[1]: https://travis-ci.org/mkdocs/mkdocs/jobs/617874568#L513